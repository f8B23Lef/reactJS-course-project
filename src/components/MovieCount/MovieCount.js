import React from 'react';
import PropTypes from 'prop-types';
import './MovieCount.scss';

export default function MovieCount({ value }) {
  return (
    <div className='count'>
      <strong>{value}</strong>
      {' '}
      movies found
    </div>
  );
}

MovieCount.propTypes = {
  value: PropTypes.number.isRequired,
};
