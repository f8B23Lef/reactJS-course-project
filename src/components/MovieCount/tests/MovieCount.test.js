import React from 'react';
import { render } from '@testing-library/react';
import MovieCount from '../MovieCount';

describe('MovieCount', () => {
  it('renders MovieCount snapshot', () => {
    const { asFragment } = render(<MovieCount value={3} />);

    expect(asFragment(<MovieCount value={3} />)).toMatchSnapshot();
  });
});
