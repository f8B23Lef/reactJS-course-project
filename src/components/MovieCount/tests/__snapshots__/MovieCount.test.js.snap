// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MovieCount renders MovieCount snapshot 1`] = `
<DocumentFragment>
  <div
    class="count"
  >
    <strong>
      3
    </strong>
     movies found
  </div>
</DocumentFragment>
`;
